frontend/src/components/browse-sightings-page/SightingCard.tsx

```diff
@@ -1,22 +1,34 @@
 import React from "react";
-import { Sighting } from "../../clients/apiClient";
+import { Sighting, SightingFromExternalApi } from "../../clients/apiClient";
 import "./BrowseSightings.scss";
 
 interface SightingProps {
-  sighting: Sighting;
+  sighting: Sighting | SightingFromExternalApi;
 }
 
 export const SightingCard: React.FunctionComponent<SightingProps> = ({
   sighting,
 }) => {
+  const imageUrl: string | undefined =
+    (sighting as Sighting).imageUrl ??
+    (sighting as SightingFromExternalApi).photoUrl;
+
+  const description: string | undefined =
+    (sighting as Sighting).description ??
+    (sighting as SightingFromExternalApi).location.description;
+
+  const whaleCount: number | undefined =
+    (sighting as Sighting).whaleCount ??
+    (sighting as SightingFromExternalApi).species.description;
```

Comment:
This doesn't make sense, because if it's an External Sighting then the card will say "Number of whales: A blue whale is a big whale!"

Reply:
or similar

Reply:
You'll need to have some conditions down in the card itself, to display different sections conditionally